public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -138,6 +150,8 @@ const getStyles = (theme: GrafanaTheme2) => ({
     text-overflow: ellipsis;
     transition: opacity 100ms linear;
     opacity: 0;
+    font-size: ${theme.typography.bodySmall.fontSize};
+    height: calc(${theme.typography.bodySmall.fontSize} + 10px);
```

Comment:
Same here. At least I personally try to use `theme.spacing()` multiples as much as possible.